lib/api/account-validation.js

```diff
@@ -1,4 +1,12 @@
-import { Meteor } from "meteor/meteor";
+/* eslint-disable no-undef */
+import {
```

Comment:
These `import` statement can be in the same line. No harm since it's just one function and it doesn't break any defined rule in ESLINT config.